src/Particular.PlatformSample.sln

```diff
@@ -11,7 +11,11 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SmokeTest", "SmokeTest\Smok
 EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{C70B4945-48B8-43A8-8518-3326D771DB89}"
 	ProjectSection(SolutionItems) = preProject
+		..\.github\workflows\ci.yml = ..\.github\workflows\ci.yml
 		Custom.Build.props = Custom.Build.props
+		..\.github\workflows\release.yml = ..\.github\workflows\release.yml
+		..\.github\workflows\stale.yml = ..\.github\workflows\stale.yml
+		..\.github\workflows\virus-scan.yml = ..\.github\workflows\virus-scan.yml
```

Comment:
stale.yml and virus-scan.yml shouldn't really be added to solution files because they're distributed by RepoStandards so we wouldn't want anyone editing them from Visual Studio anyway.